apps/prairielearn/src/tests/manualGrading.test.ts

```diff
@@ -1160,14 +1181,16 @@ describe('Manual Grading', { timeout: 80_000 }, function () {
         'manual grading page for assessment question should list one instance',
         async () => {
           setUser(defaultUser);
-          const manualGradingAQData = await (
-            await fetch(manualGradingAssessmentQuestionUrl + '/instances.json')
-          ).text();
-          const instanceList = JSON.parse(manualGradingAQData)?.instance_questions;
+          const manualGradingAQData = (await (
+            await fetch(manualGradingAssessmentQuestionUrl + '/instances.json', {
+              headers: { Accept: 'application/json' },
+            })
+          ).json()) as { instance_questions: InstanceQuestionRowWithAIGradingStats[] };
```

Comment:
Not requesting a change in this PR: our "API" is really more RPC than an API, I'm wondering if we should be using `tRPC` for client->server interactions? Mainly for the strong typing, it seems verrrrry attractive.

Reply:
+1 on tRPC.